Rooms/familyRoom.json

```diff
@@ -15,12 +15,14 @@
     },
     "verbInteractions": {
         "Examine": {
+            "walls": "The paint is a rich green. Your favorite color.",
+            "floor": "The floor is a nice dark hardwood, the same as most of the house.",
             "shelves": "It's a heavy wooden shelving unit, made to hold entertainment equipment. There's not much in it yet.",
             "boxes": "You glance through the moving boxes, but there's nothing useful in here.",
             "tv": "You wish you could believe it was just the TV making noise in the house, but that seems unlikely with the power being out.",
             "rack": "It's just a simple metal pole. Your jacket is hanging from it.",
             "jacket": "It's mostly fake leather. There is nothing in the pockets.",
-            "hooks": "You glance through the moving boxes, but there's nothing useful in here.",
+            "hooks": "There are four hooks screwed into the wall. One of them is holding a key.",
             "key": "You glance through the moving boxes, but there's nothing useful in here."

```

Comment:
I'm not sure when this happened, but Examine["key"] doesn't appear to be correct.